src/main/resources/edu/cornell/kfs/concur/cu-spring-concur.xml

```diff
@@ -531,6 +531,7 @@
         <property name="transactionsBypassedLabel" value="Bypassed Expense Type XXXX Transactions :"/>
         <property name="pdpRecordsProcessedLabel" value="PDP Records Processed :"/>
         <property name="reportValidationErrorsSubTitle" value="Concur Standard Accounting Extract Records With Validation Errors"/>
+        <property name="reportValidationErrorsSubTitleByPassedTransactionNote" value="COPD (Corporate Paid) and Expense Type XXXX transactions are information only and do not represent errors"/>
```

Comment:
I believe there are some cases where an "XXXX" line could still be processed, such as when it's related to an ATM Cash Advance. If practical, it might be a good idea to have this message reflect that.

Reply:
I will ping Cathy and Tammy on this, I don't have a good idea on what it should say.